src/libs/colorpicker.c

```diff
@@ -268,17 +268,17 @@ static void _update_samples_output(dt_lib_module_t *self)
     }
 
     // Setting the output button
-    sample->rgb.red = rgb[0] / 255.0;
-    sample->rgb.green = rgb[1] / 255.0;
-    sample->rgb.blue = rgb[2] / 255.0;
+    sample->rgb.red = rgb[0];
```

Comment:
Why don't you CLAMP the values here? As done in line 181. Seems like it is needed here too, no?

Reply:
I'm clamping only when I assign it to the color widget, I want to display the actual pixel values, even when out of range.

Reply:
But that's what happen here no? You queue the update for the color_patch just below. Those are for the live samples which happen also to have a widget to display the color for the given area.

Reply:
Ups, you're right, its fixed now. Thanks!